Lib/annotationlib.py

```diff
@@ -45,7 +45,17 @@ class Format(enum.IntEnum):
 
 
 class ForwardRef:
-    """Wrapper that holds a forward reference."""
+    """Wrapper that holds a forward reference.
+
+    Constructor arguments:
+    - arg: a string representing the code to be evaluated.
+    - module: the module where the forward reference was created. Must be a string,
+      not a module object.
+    - owner: The owning object (module, class, or function).
+    - is_argument: Does nothing, retained for compatibility.
+    - is_class: True if the forward reference was created in class scope.
```

Comment:
Occasionally we get complaints from downstream users because their docs generation tools pick up docstrings from CPython that turn out to be invalid ReST. I don't feel like we should go out of our way to support those users, but we could at least use asterisks instead of hyphens:

```suggestion
    * arg: a string representing the code to be evaluated.
    * module: the module where the forward reference was created.
      Must be a string, not a module object.
    * owner: The owning object (module, class, or function).
    * is_argument: Does nothing, retained for compatibility.
    * is_class: True if the forward reference was created in class scope.
```